services-api/src/main/java/io/scalecube/services/Microservices.java

```diff
@@ -0,0 +1,15 @@
+package io.scalecube.services;
+
+import io.scalecube.net.Address;
+import io.scalecube.services.discovery.api.ServiceDiscovery;
+
+public interface Microservices {
```

Comment:
Sounds great. But there's one thing. ServiceFactory (ex ServicesProvider) is in the service-api module, which is quite reasonable. And Scalecube (ex Microservices) is in the implementation module, spring-services. I do not think it is worth creating service-api => scalecube-services.

Moreover, the Microservices interface fully defines the functionality of the old Microservices (now Scalecube), not a part of it. 

The only thing that I would think about is choosing names because of backward compatibility. Keep the old name for the Microservices implementation, and rename the interface